src/bitte_ci/runner.cr

```diff
@@ -146,6 +156,47 @@ module BitteCI
       end
     end
 
+    def fetch_devshell_steps_json
+      full_name = @pr.base.repo.full_name
+      sha = @pr.base.sha
+
+      
+
+      entrypoint_mem = IO::Memory.new
+      status = Process.run("nix", output: entrypoint_mem, error: STDERR,
+        args: ["eval", "--raw", "github:#{full_name}/#{sha}.#devShell.x86_64-linux.outPath"])
```

Comment:
How do we get the correct `system` from this environment?

Reply:
would a call to `nix eval --impure --raw --expr "builtins.currentSystem"` work?

Reply:
Sounds good! Thx